display_list/display_list_ops.h

```diff
@@ -269,6 +269,24 @@ struct SetRuntimeEffectColorSourceOp : DLOp {
   }
 };
 
+struct SetSharedColorFilterOp : DLOp {
```

Comment:
A comment on this struct about its memory usage in the buffer, similar to the other "SetShared" Ops.